packages/itwin/ec3-widget/src/components/EC3/EC3Config.ts

```diff
@@ -20,6 +29,7 @@ export interface EC3ConfigCommonProps {
    */
   clientId: string;
   iTwinId: string;
+  iModelId: string;
```

Comment:
Why is iModelId needed here? If it is really needed - this is a breaking change, therefore this is no longer a patch.

Reply:
Removed dependency on iModelId